features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/ServiceCatalog.js

```diff
@@ -0,0 +1,277 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable */
+import cloneDeep from 'lodash.clonedeep';
+import Utils from './Utils';
+import {
+    getSettings,
+    searchServices,
+    checkServiceExistence,
```

Comment:
Shall we move these methods into a class as static methods, So if in the future if we add a new method to mock we don't need to change/update the import statements

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/9372/commits/929acf282539bdf1dd875f64e98ceec5152609c3